lapis/src/main/kotlin/org/genspectrum/lapis/model/VariantQueryCustomListener.kt

```diff
@@ -141,8 +343,13 @@ class VariantQueryCustomListener(
         if (ctx == null) {
             return
         }
+        try {
+            referenceGenomeSchema.getGeneFromLowercaseName(ctx.geneOrName().text.lowercase()).name
```

Comment:
This is an issue of the existing code: I think `getGeneFromLowercaseName` should be renamed to something like "get gene ignoring case" and hide the `.lowercase()` inside. Then the error message would be better, too.

Reply:
I will make a small PR for this change